scripts/semconv/templates/common.j2

```diff
@@ -24,3 +24,12 @@
 from deprecated import deprecated
     {%- endif %}
 {%- endmacro-%}
+
+{%- macro add_and_check_constant(fqn, const_name, all_names) -%}
+{%- if const_name in all_names -%}
+!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
```

Comment:
check final annotation and check if mypy runs on this code

Reply:
https://github.com/open-telemetry/opentelemetry-python/blob/main/tox.ini#L196

Looks like we currently only run type checking on the api package.

Reply:
thanks! let me try to repeat it for semconv

Reply:
added and tried if it checks duplicates (it does!)

opentelemetry-semantic-conventions/src/opentelemetry/semconv/_incubating/attributes/messaging_attributes.py:29: error: Cannot redefine an existing name as final  [misc]